knowledge-base/grid-dynamically-updating-filtermenutemplate-value.md

```diff
@@ -0,0 +1,208 @@
+---
+title: FilterMenuTemplate Not Updating After  Dynamic Value Change
```

Comment:
Use "FilterMenuTemplate" and "Filter Menu Template" in similar quantities in the KB for SEO purposes.
```suggestion
title: FilterMenuTemplate Not Updating After Dynamic Value Change
```